src/test/java/org/folio/innreach/controller/CentralServerControllerTest.java

```diff
@@ -147,7 +147,10 @@ void return404HttpCode_when_updatableCentralServerNotFound() {
   }
 
   @Test
-  @Sql(scripts = "classpath:db/central-server/pre-populate-central-server.sql")
+  @Sql(scripts = {
+    "classpath:db/central-server/pre-populate-central-server.sql",
+    "classpath:db/itm-contrib-opt-conf/pre-populate-itm-contrib-opt-conf.sql"
```

Comment:
For Central Server Controller item contribution options are not relevant.
Please adjust the code and make item contribution options optional